ros2topic/ros2topic/verb/pub.py

```diff
@@ -69,19 +69,28 @@ def add_arguments(self, parser, cli_name):
         parser.add_argument(
             '-n', '--node-name',
             help='Name of the created publishing node')
+        parser.add_argument(
+            '-t', '--times', type=int, default=0,
+            help='Publish this number of times and then exit')
         add_qos_arguments_to_argument_parser(
             parser, is_publisher=True, default_preset='system_default')
 
     def main(self, *, args):
         if args.rate <= 0:
             raise RuntimeError('rate must be greater than zero')
 
+        if args.once and args.times > 0:
```

Comment:
```suggestion
        if args.once and args.times > 1:
```

i think that with once and times == 1, we could proceed.

Reply:
While that is true, and would be logically consistent, in general passing both `-1` and `-t` isn't logically consistent.  Thus, I would rather always throw an error here saying "you can't pass both of these at the same time".